js/apps/system/_admin/aardvark/APP/react/src/views/query/editor/ImportQueryModal.tsx

```diff
@@ -66,20 +66,11 @@ export const ImportQueryModal = ({
       reader.readAsText(file);
       reader.onload = async () => {
         const data = reader.result;
-        if (typeof data !== "string") {
-          return;
-        }
+        if (typeof data !== "string") return;
         const queries = JSON.parse(data);
-        const sanitizedQueries = queries.map((query: QueryType) => {
-          return {
-            name: query.name,
-            value: query.value,
-            parameter: query.parameter
-          };
-        });
```

Comment:
@pluma4345 this probably sneaked in while resolving merge conflicts? This should be reverted